### PR TITLE
Added missing HostToken to AirPricingSolution_XML

### LINKED ISSUE #412: Receiving "The following attribute is missing in the AirCreateReservationReq: AirPricingSolution/HostToken." despite the req payload including an AirPricingSolution element
Hi there,

For some reason I am suddenly getting "The following attribute is missing in the AirCreateReservationReq: AirPricingSolution/HostToken." in the response from AirCreateReservation even though the element is included in the request. This causes ticketing to fail for the generated PNR. Req and Res are below:

```
<soap:Envelope
  xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
  <soap:Header
    xmlns:univ="http://www.travelport.com/schema/universal_v47_0">
    <univ:SupportedVersions airVersion="air_v47_0"/>
  </soap:Header>
  <soap:Body>
    <univ:AirCreateReservationReq
            AuthorizedBy="user" TraceId=""
            RetainReservation="Both" TargetBranch="P7118380"
            RuleName="SIP"

            RestrictWaitlist="false"

      xmlns:univ="http://www.travelport.com/schema/universal_v47_0"

      xmlns:com="http://www.travelport.com/schema/common_v47_0"

      xmlns:air="http://www.travelport.com/schema/air_v47_0"

      xmlns:common_v47_0="http://www.travelport.com/schema/common_v47_0"
            >
      <com:BillingPointOfSaleInfo OriginApplication="uAPI" />
      <com:BookingTraveler Key="P_0" Age="50" DOB="1968-07-25" Gender="M" TravelerType="ADT">
        <com:BookingTravelerName First="SKYWALKER" Last="ANAKIN" />
        <com:PhoneNumber
                            CountryCode="38"
                            Location="IEV"
                            Number="0660419905"
                            Type="Other"
                    />
        <com:SSR Type="DOCS" FreeText="P/UA/ES221731/UA/25Jul68/M/13Jun20/ANAKIN/SKYWALKER"/>
        <!--<com:SSR Carrier="F9" FreeText="P/IN/F1234567/IN/05Jan85/M/13Dec14/Jones/Stephen"-->
        <!--SegmentRef="IVaa0tGWQ+2grqSIL81xPQ==" Status="HK" Type="DOCS"/>-->
      </com:BookingTraveler>
      <air:AirPricingSolution Key="XWjoMt3R2BKAQ/goAAAAAA==" TotalPrice="USD313.10" BasePrice="GBP41.00" ApproximateTotalPrice="USD313.10" ApproximateBasePrice="USD52.00" EquivalentBasePrice="USD52.00" Taxes="USD261.10" ApproximateTaxes="USD261.10" QuoteDate="2019-06-13" >
        <air:AirSegment Key="XWjoMt3R2BKAI/goAAAAAA==" Group="0" Carrier="AZ" FlightNumber="207" ProviderCode="1V" Origin="LHR" Destination="FCO" DepartureTime="2020-02-10T16:55:00.000+00:00" ArrivalTime="2020-02-10T20:30:00.000+01:00" FlightTime="155" TravelTime="155" Distance="894" ClassOfService="X" Equipment="321" ChangeOfPlane="false" OptionalServicesIndicator="true" LinkAvailability="true" AvailabilityDisplayType="Fare Specific Fare Quote Unbooked">
          <air:CodeshareInfo OperatingCarrier="AZ">Alitalia Societa Aerea Italiana SPA</air:CodeshareInfo>
          <air:FlightDetails Key="XWjoMt3R2BKAJ/goAAAAAA==" Origin="LHR" Destination="FCO" DepartureTime="2020-02-10T16:55:00.000+00:00" ArrivalTime="2020-02-10T20:30:00.000+01:00" FlightTime="155" TravelTime="155" Equipment="321" OriginTerminal="4" DestinationTerminal="3" AutomatedCheckin="false">
            <air:Meals>ComplimentaryRefreshments</air:Meals>
            <air:InFlightServices>Non-smoking</air:InFlightServices>
          </air:FlightDetails>
          <air:Connection/>
        </air:AirSegment>
        <air:AirSegment Key="XWjoMt3R2BKAK/goAAAAAA==" Group="0" Carrier="AZ" FlightNumber="332" ProviderCode="1V" Origin="FCO" Destination="CDG" DepartureTime="2020-02-10T21:15:00.000+01:00" ArrivalTime="2020-02-10T23:25:00.000+01:00" FlightTime="130" TravelTime="130" Distance="686" ClassOfService="X" Equipment="320" ChangeOfPlane="false" OptionalServicesIndicator="true" LinkAvailability="true" AvailabilityDisplayType="Fare Specific Fare Quote Unbooked">
          <air:CodeshareInfo OperatingCarrier="AZ">Alitalia Societa Aerea Italiana SPA</air:CodeshareInfo>
          <air:FlightDetails Key="XWjoMt3R2BKAL/goAAAAAA==" Origin="FCO" Destination="CDG" DepartureTime="2020-02-10T21:15:00.000+01:00" ArrivalTime="2020-02-10T23:25:00.000+01:00" FlightTime="130" TravelTime="130" Equipment="320" OriginTerminal="1" DestinationTerminal="2F" AutomatedCheckin="false">
            <air:Meals>ComplimentaryRefreshments</air:Meals>
            <air:InFlightServices>Non-smoking</air:InFlightServices>
          </air:FlightDetails>
        </air:AirSegment>
        <air:AirSegment Key="XWjoMt3R2BKAM/goAAAAAA==" Group="1" Carrier="AZ" FlightNumber="325" ProviderCode="1V" Origin="CDG" Destination="FCO" DepartureTime="2020-02-15T18:20:00.000+01:00" ArrivalTime="2020-02-15T20:25:00.000+01:00" FlightTime="125" TravelTime="125" Distance="686" ClassOfService="X" Equipment="321" ChangeOfPlane="false" OptionalServicesIndicator="true" LinkAvailability="true" AvailabilityDisplayType="Fare Specific Fare Quote Unbooked">
          <air:CodeshareInfo OperatingCarrier="AZ">Alitalia Societa Aerea Italiana SPA</air:CodeshareInfo>
          <air:FlightDetails Key="XWjoMt3R2BKAN/goAAAAAA==" Origin="CDG" Destination="FCO" DepartureTime="2020-02-15T18:20:00.000+01:00" ArrivalTime="2020-02-15T20:25:00.000+01:00" FlightTime="125" TravelTime="125" Equipment="321" OriginTerminal="2F" DestinationTerminal="1" AutomatedCheckin="false">
            <air:Meals>ComplimentaryRefreshments</air:Meals>
            <air:InFlightServices>Non-smoking</air:InFlightServices>
          </air:FlightDetails>
          <air:Connection/>
        </air:AirSegment>
        <air:AirSegment Key="XWjoMt3R2BKAO/goAAAAAA==" Group="1" Carrier="AZ" FlightNumber="210" ProviderCode="1V" Origin="FCO" Destination="LHR" DepartureTime="2020-02-15T21:25:00.000+01:00" ArrivalTime="2020-02-15T23:05:00.000+00:00" FlightTime="160" TravelTime="160" Distance="894" ClassOfService="X" Equipment="32S" ChangeOfPlane="false" OptionalServicesIndicator="true" LinkAvailability="true" AvailabilityDisplayType="Fare Specific Fare Quote Unbooked">
          <air:CodeshareInfo OperatingCarrier="AZ">Alitalia Societa Aerea Italiana SPA</air:CodeshareInfo>
          <air:FlightDetails Key="XWjoMt3R2BKAP/goAAAAAA==" Origin="FCO" Destination="LHR" DepartureTime="2020-02-15T21:25:00.000+01:00" ArrivalTime="2020-02-15T23:05:00.000+00:00" FlightTime="160" TravelTime="160" Equipment="32S" OriginTerminal="1" DestinationTerminal="4" AutomatedCheckin="false">
            <air:Meals>ComplimentaryRefreshments</air:Meals>
            <air:InFlightServices>Non-smoking</air:InFlightServices>
          </air:FlightDetails>
        </air:AirSegment>
        <air:AirPricingInfo Key="XWjoMt3R2BKAT/goAAAAAA==" TotalPrice="USD313.10" BasePrice="GBP41.00" ApproximateTotalPrice="USD313.10" ApproximateBasePrice="USD52.00" EquivalentBasePrice="USD52.00" Taxes="USD261.10" LatestTicketingTime="2020-02-10T23:59:00.000-08:00" PricingMethod="Auto" IncludesVAT="false" ETicketability="Yes" PlatingCarrier="AZ" ProviderCode="1V">
          <air:FareInfo Key="XWjoMt3R2BKAf/goAAAAAA==" FareBasis="XRLGEU1" PassengerTypeCode="ADT" Origin="LHR" Destination="CDG" EffectiveDate="2019-06-13T03:53:00.000-07:00" DepartureDate="2020-02-10" Amount="USD26.00" NotValidBefore="2020-02-10" NotValidAfter="2020-02-10" TaxAmount="USD136.70">
            <common_v47_0:Endorsement Value="AZ ONLY"/>
            <air:FareRuleKey FareInfoRef="XWjoMt3R2BKAf/goAAAAAA==" ProviderCode="1V">6UUVoSldxwjW7C5aEBkYisbKj3F8T9EyxsqPcXxP0TLGyo9xfE/RMsuWFfXVd1OAly5qxZ3qLwOXLmrFneovA5cuasWd6i8Dly5qxZ3qLwOXLmrFneovAxcUEzjBLEuJ4bHmxr88+l3zd/VTAJtvZUn29rtRX7RVpN3NI/xFQ21+CeOefa9UvDA7PpK/EK2mvJWr6+nvtEKK4CyywU9wncCIbRqMTJch6v9tEaRJgF5C/YIEuJEeltKd89PywREqGXWl46AEslMqfYwZEkBLigLWhthMb3jlNac+Hza997l2xAv9znkmMUKqqnCgnUKjly5qxZ3qLwOXLmrFneovA5cuasWd6i8Dly5qxZ3qLwOXLmrFneovA2+QKPIWaRvq6oxE1UL944DjaSte7T4ddDlN7mTMg9NhjT6I8YN9eGPrqN84gBC4jx4WExVb316nosmMaNBEpBo=</air:FareRuleKey>
            <air:Brand Key="XWjoMt3R2BKAf/goAAAAAA==" BrandFound="false" UpSellBrandFound="false"/>
          </air:FareInfo>
          <air:FareInfo Key="XWjoMt3R2BKAh/goAAAAAA==" FareBasis="XRLGEU1" PassengerTypeCode="ADT" Origin="CDG" Destination="LHR" EffectiveDate="2019-06-13T03:53:00.000-07:00" DepartureDate="2020-02-15" Amount="USD26.00" NotValidBefore="2020-02-15" NotValidAfter="2020-02-15" TaxAmount="USD124.40">
            <common_v47_0:Endorsement Value="AZ ONLY"/>
            <air:FareRuleKey FareInfoRef="XWjoMt3R2BKAh/goAAAAAA==" ProviderCode="1V">6UUVoSldxwjW7C5aEBkYisbKj3F8T9EyxsqPcXxP0TLGyo9xfE/RMsuWFfXVd1OAly5qxZ3qLwOXLmrFneovA5cuasWd6i8Dly5qxZ3qLwOXLmrFneovAxcUEzjBLEuJ4bHmxr88+l3zd/VTAJtvZSix0LjmmjI1ScgwocIH/TR+CeOefa9UvDA7PpK/EK2mvJWr6+nvtELlaLg4V03z98CIbRqMTJch6v9tEaRJgF5C/YIEuJEeltKd89PywREqGXWl46AEslMWDUeWSpAMCwLWhthMb3jliNoOYi19ZGt2xAv9znkmMUKqqnCgnUKjly5qxZ3qLwOXLmrFneovA5cuasWd6i8Dly5qxZ3qLwOXLmrFneovA2+QKPIWaRvq6oxE1UL944DjaSte7T4ddDlN7mTMg9NhjT6I8YN9eGPrqN84gBC4jx4WExVb316nosmMaNBEpBo=</air:FareRuleKey>
            <air:Brand Key="XWjoMt3R2BKAh/goAAAAAA==" BrandFound="false" UpSellBrandFound="false"/>
          </air:FareInfo>
          <air:BookingInfo BookingCode="X" CabinClass="Economy" FareInfoRef="XWjoMt3R2BKAf/goAAAAAA==" SegmentRef="XWjoMt3R2BKAI/goAAAAAA==" HostTokenRef="XWjoMt3R2BKAR/goAAAAAA=="/>
          <air:BookingInfo BookingCode="X" CabinClass="Economy" FareInfoRef="XWjoMt3R2BKAf/goAAAAAA==" SegmentRef="XWjoMt3R2BKAK/goAAAAAA==" HostTokenRef="XWjoMt3R2BKAR/goAAAAAA=="/>
          <air:BookingInfo BookingCode="X" CabinClass="Economy" FareInfoRef="XWjoMt3R2BKAh/goAAAAAA==" SegmentRef="XWjoMt3R2BKAM/goAAAAAA==" HostTokenRef="XWjoMt3R2BKAS/goAAAAAA=="/>
          <air:BookingInfo BookingCode="X" CabinClass="Economy" FareInfoRef="XWjoMt3R2BKAh/goAAAAAA==" SegmentRef="XWjoMt3R2BKAO/goAAAAAA==" HostTokenRef="XWjoMt3R2BKAS/goAAAAAA=="/>
          <air:TaxInfo Category="GB" Amount="USD16.40" Key="XWjoMt3R2BKAU/goAAAAAA=="/>
          <air:TaxInfo Category="UB" Amount="USD25.80" Key="XWjoMt3R2BKAV/goAAAAAA=="/>
          <air:TaxInfo Category="HB" Amount="USD16.80" Key="XWjoMt3R2BKAW/goAAAAAA=="/>
          <air:TaxInfo Category="IT" Amount="USD13.40" Key="XWjoMt3R2BKAX/goAAAAAA=="/>
          <air:TaxInfo Category="MJ" Amount="USD2.20" Key="XWjoMt3R2BKAY/goAAAAAA=="/>
          <air:TaxInfo Category="VT" Amount="USD3.60" Key="XWjoMt3R2BKAZ/goAAAAAA=="/>
          <air:TaxInfo Category="FR" Amount="USD18.90" Key="XWjoMt3R2BKAa/goAAAAAA=="/>
          <air:TaxInfo Category="IZ" Amount="USD1.30" Key="XWjoMt3R2BKAb/goAAAAAA=="/>
          <air:TaxInfo Category="QX" Amount="USD13.30" Key="XWjoMt3R2BKAc/goAAAAAA=="/>
          <air:TaxInfo Category="YQ" Amount="USD6.60" Key="XWjoMt3R2BKAd/goAAAAAA=="/>
          <air:TaxInfo Category="YR" Amount="USD142.80" Key="XWjoMt3R2BKAe/goAAAAAA=="/>
          <air:FareCalc>LON AZ X/ROM AZ PAR 26.81XRLGEU1 AZ X/ROM AZ LON 26.81XRLGEU1 NUC53.62END ROE0.764586</air:FareCalc>
          <air:PassengerType BookingTravelerRef="P_0" Code="ADT" Age="50"/>
          <air:ChangePenalty PenaltyApplies="Anytime">
            <air:Percentage>100.00</air:Percentage>
          </air:ChangePenalty>
          <air:CancelPenalty PenaltyApplies="Anytime">
            <air:Percentage>100.00</air:Percentage>
          </air:CancelPenalty>
          <air:BaggageAllowances>
            <air:BaggageAllowanceInfo TravelerType="ADT" Origin="LON" Destination="PAR" Carrier="AZ">
              <air:URLInfo>
                <air:URL>VIEWTRIP.TRAVELPORT.COM/BAGGAGEPOLICY/AZ</air:URL>
              </air:URLInfo>
              <air:TextInfo>
                <air:Text>0P</air:Text>
                <air:Text>BAGGAGE DISCOUNTS MAY APPLY BASED ON FREQUENT FLYER STATUS/ ONLINE CHECKIN/FORM OF PAYMENT/MILITARY/ETC.</air:Text>
              </air:TextInfo>
              <air:BagDetails ApplicableBags="1stChecked">
                <air:BaggageRestriction>
                  <air:TextInfo>
                    <air:Text>BAGGAGE CHARGES DATA NOT AVAILABLE</air:Text>
                  </air:TextInfo>
                </air:BaggageRestriction>
              </air:BagDetails>
              <air:BagDetails ApplicableBags="2ndChecked">
                <air:BaggageRestriction>
                  <air:TextInfo>
                    <air:Text>BAGGAGE CHARGES DATA NOT AVAILABLE</air:Text>
                  </air:TextInfo>
                </air:BaggageRestriction>
              </air:BagDetails>
            </air:BaggageAllowanceInfo>
            <air:BaggageAllowanceInfo TravelerType="ADT" Origin="PAR" Destination="LON" Carrier="AZ">
              <air:URLInfo>
                <air:URL>VIEWTRIP.TRAVELPORT.COM/BAGGAGEPOLICY/AZ</air:URL>
              </air:URLInfo>
              <air:TextInfo>
                <air:Text>0P</air:Text>
                <air:Text>BAGGAGE DISCOUNTS MAY APPLY BASED ON FREQUENT FLYER STATUS/ ONLINE CHECKIN/FORM OF PAYMENT/MILITARY/ETC.</air:Text>
              </air:TextInfo>
              <air:BagDetails ApplicableBags="1stChecked">
                <air:BaggageRestriction>
                  <air:TextInfo>
                    <air:Text>BAGGAGE CHARGES DATA NOT AVAILABLE</air:Text>
                  </air:TextInfo>
                </air:BaggageRestriction>
              </air:BagDetails>
              <air:BagDetails ApplicableBags="2ndChecked">
                <air:BaggageRestriction>
                  <air:TextInfo>
                    <air:Text>BAGGAGE CHARGES DATA NOT AVAILABLE</air:Text>
                  </air:TextInfo>
                </air:BaggageRestriction>
              </air:BagDetails>
            </air:BaggageAllowanceInfo>
            <air:CarryOnAllowanceInfo Origin="LON" Destination="ROM" Carrier="AZ">
              <air:TextInfo>
                <air:Text>BAGGAGE ALLOWANCE DATA NOT AVAILABLE</air:Text>
              </air:TextInfo>
            </air:CarryOnAllowanceInfo>
            <air:CarryOnAllowanceInfo Origin="ROM" Destination="PAR" Carrier="AZ">
              <air:TextInfo>
                <air:Text>BAGGAGE ALLOWANCE DATA NOT AVAILABLE</air:Text>
              </air:TextInfo>
            </air:CarryOnAllowanceInfo>
            <air:CarryOnAllowanceInfo Origin="PAR" Destination="ROM" Carrier="AZ">
              <air:TextInfo>
                <air:Text>BAGGAGE ALLOWANCE DATA NOT AVAILABLE</air:Text>
              </air:TextInfo>
            </air:CarryOnAllowanceInfo>
            <air:CarryOnAllowanceInfo Origin="ROM" Destination="LON" Carrier="AZ">
              <air:TextInfo>
                <air:Text>BAGGAGE ALLOWANCE DATA NOT AVAILABLE</air:Text>
              </air:TextInfo>
            </air:CarryOnAllowanceInfo>
          </air:BaggageAllowances>
        </air:AirPricingInfo>
        <air:FareNote Key="XWjoMt3R2BKAn/goAAAAAA==">SUM IDENTIFIED AS UB IS A PASSENGER SERVICE CHARGE</air:FareNote>
        <air:FareNote Key="XWjoMt3R2BKAo/goAAAAAA==">RATE USED IN EQU TOTAL IS BSR 1GBP - 1.264273USD</air:FareNote>
        <air:FareNote Key="XWjoMt3R2BKAp/goAAAAAA==">LAST DATE TO PURCHASE TICKET: 10FEB20</air:FareNote>
        <air:FareNote Key="XWjoMt3R2BKAq/goAAAAAA==">TICKETING AGENCY 1D4M</air:FareNote>
        <air:FareNote Key="XWjoMt3R2BKAr/goAAAAAA==">DEFAULT PLATING CARRIER AZ</air:FareNote>
        <air:FareNote Key="XWjoMt3R2BKAs/goAAAAAA==">FARE HAS A PLATING CARRIER RESTRICTION</air:FareNote>
        <air:FareNote Key="XWjoMt3R2BKAt/goAAAAAA==">E-TKT REQUIRED</air:FareNote>
      </air:AirPricingSolution>
      <com:ActionStatus Type="TAU" TicketDate="2019-06-13T14:53:12+01:00" ProviderCode="1V" />
    </univ:AirCreateReservationReq>
  </soap:Body>
</soap:Envelope>
```

```
<SOAP:Envelope
  xmlns:SOAP="http://schemas.xmlsoap.org/soap/envelope/">
  <SOAP:Body>
    <universal:AirCreateReservationRsp TraceId="" TransactionId="50785A520A07643BABE3687F1BD925A9" ResponseTime="2693"
      xmlns:universal="http://www.travelport.com/schema/universal_v47_0"
      xmlns:common_v47_0="http://www.travelport.com/schema/common_v47_0"
      xmlns:air="http://www.travelport.com/schema/air_v47_0">
      <common_v47_0:ResponseMessage Code="0" Type="Warning" ProviderCode="1V">NO STORED FARES EXIST</common_v47_0:ResponseMessage>
      <common_v47_0:ResponseMessage Code="0" Type="Warning" ProviderCode="1V">Some of the requested AirPricingInfos could not be saved for the requested provider.</common_v47_0:ResponseMessage>
      <common_v47_0:ResponseMessage Code="0" Type="Error" ProviderCode="1V">The following attribute is missing in the AirCreateReservationReq: AirPricingSolution/HostToken.</common_v47_0:ResponseMessage>
      <universal:UniversalRecord LocatorCode="X1YXTY" Version="0" Status="Active">
        <common_v47_0:BookingTraveler Key="bGwfMt7Q2BKAxFVrAAAAAA==" TravelerType="ADT" Age="50" DOB="1968-07-25" Gender="M" ElStat="A">
          <common_v47_0:BookingTravelerName First="SKYWALKER" Last="ANAKIN"/>
          <common_v47_0:PhoneNumber Key="bGwfMt7Q2BKAyFVrAAAAAA==" Type="Other" Location="IEV" CountryCode="38" Number="0660419905" ElStat="A">
            <common_v47_0:ProviderReservationInfoRef Key="bGwfMt7Q2BKArMVrAAAAAA=="/>
          </common_v47_0:PhoneNumber>
          <common_v47_0:SSR Key="bGwfMt7Q2BKAzFVrAAAAAA==" Status="HK" Type="DOCS" FreeText="P/UA/00001731/UA/0000000/M/13JUN20/ANAKIN/SKYWALKER-1ANAKIN/SKYWALKER" Carrier="AZ" ProviderReservationInfoRef="bGwfMt7Q2BKArMVrAAAAAA==" ElStat="A"/>
        </common_v47_0:BookingTraveler>
        <common_v47_0:ActionStatus Key="bGwfMt7Q2BKA8FVrAAAAAA==" Type="TAU" TicketDate="2019-06-13T23:59:00.000-07:00" ProviderReservationInfoRef="bGwfMt7Q2BKArMVrAAAAAA==" ProviderCode="1V" ElStat="A"/>
        <universal:ProviderReservationInfo Key="bGwfMt7Q2BKArMVrAAAAAA==" ProviderCode="1V" LocatorCode="M8LFHI" CreateDate="2019-06-13T10:53:16.429+00:00" ModifiedDate="2019-06-13T10:53:16.429+00:00" HostCreateDate="2019-06-13" OwningPCC="1D4M"/>
        <air:AirReservation LocatorCode="KU1U71" CreateDate="2019-06-13T10:53:16.142+00:00" ModifiedDate="2019-06-13T10:53:16.429+00:00">
          <common_v47_0:SupplierLocator SupplierCode="AZ" SupplierLocatorCode="KOTOGQ" ProviderReservationInfoRef="bGwfMt7Q2BKArMVrAAAAAA==" CreateDateTime="2019-06-13T10:53:00.000+00:00"/>
          <common_v47_0:BookingTravelerRef Key="bGwfMt7Q2BKAxFVrAAAAAA=="/>
          <common_v47_0:ProviderReservationInfoRef Key="bGwfMt7Q2BKArMVrAAAAAA=="/>
          <air:AirSegment Key="XWjoMt3R2BKAI/goAAAAAA==" Group="0" Carrier="AZ" CabinClass="Economy" FlightNumber="207" ProviderCode="1V" Origin="LHR" Destination="FCO" DepartureTime="2020-02-10T16:55:00.000+00:00" ArrivalTime="2020-02-10T20:30:00.000+01:00" TravelTime="155" ClassOfService="X" ETicketability="Yes" Equipment="321" MarriageGroup="1" Status="HK" ChangeOfPlane="false" GuaranteedPaymentCarrier="No" ProviderReservationInfoRef="bGwfMt7Q2BKArMVrAAAAAA==" TravelOrder="1" ProviderSegmentOrder="1" OptionalServicesIndicator="false" LinkAvailability="true" ElStat="A">
            <air:FlightDetails Key="XWjoMt3R2BKAJ/goAAAAAA==" Origin="LHR" Destination="FCO" DepartureTime="2020-02-10T16:55:00.000+00:00" ArrivalTime="2020-02-10T20:30:00.000+01:00" FlightTime="155" TravelTime="155" Equipment="321" OriginTerminal="4" DestinationTerminal="3" AutomatedCheckin="false" ElStat="A"/>
            <air:Connection Duration="45"/>
            <common_v47_0:SellMessage>DEPARTS LHR TERMINAL 4  - ARRIVES FCO TERMINAL 3</common_v47_0:SellMessage>
          </air:AirSegment>
          <air:AirSegment Key="XWjoMt3R2BKAK/goAAAAAA==" Group="0" Carrier="AZ" CabinClass="Economy" FlightNumber="332" ProviderCode="1V" Origin="FCO" Destination="CDG" DepartureTime="2020-02-10T21:15:00.000+01:00" ArrivalTime="2020-02-10T23:25:00.000+01:00" TravelTime="130" ClassOfService="X" ETicketability="Yes" Equipment="320" MarriageGroup="1" Status="HK" ChangeOfPlane="false" GuaranteedPaymentCarrier="No" ProviderReservationInfoRef="bGwfMt7Q2BKArMVrAAAAAA==" TravelOrder="2" ProviderSegmentOrder="2" OptionalServicesIndicator="false" LinkAvailability="true" ElStat="A">
            <air:FlightDetails Key="bGwfMt7Q2BKAmMVrAAAAAA==" Origin="FCO" Destination="CDG" DepartureTime="2020-02-10T21:15:00.000+01:00" ArrivalTime="2020-02-10T23:25:00.000+01:00" FlightTime="130" TravelTime="130" Equipment="320" OriginTerminal="1" DestinationTerminal="2F" AutomatedCheckin="false" ElStat="C"/>
            <common_v47_0:SellMessage>OFFER CAR/HOTEL    >CAL	     >HOA</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>DEPARTS FCO TERMINAL 1  - ARRIVES CDG TERMINAL 2F</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>ADD ADVANCE PASSENGER INFORMATION SSRS DOCA/DOCO/DOCS</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>PERSONAL DATA WHICH IS PROVIDED TO US IN CONNECTION</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>WITH YOUR TRAVEL MAY BE PASSED TO GOVERNMENT AUTHORITIES</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>FOR BORDER CONTROL AND AVIATION SECURITY PURPOSES</common_v47_0:SellMessage>
          </air:AirSegment>
          <air:AirSegment Key="XWjoMt3R2BKAM/goAAAAAA==" Group="1" Carrier="AZ" CabinClass="Economy" FlightNumber="325" ProviderCode="1V" Origin="CDG" Destination="FCO" DepartureTime="2020-02-15T18:20:00.000+01:00" ArrivalTime="2020-02-15T20:25:00.000+01:00" TravelTime="125" ClassOfService="X" ETicketability="Yes" Equipment="321" MarriageGroup="2" Status="HK" ChangeOfPlane="false" GuaranteedPaymentCarrier="No" ProviderReservationInfoRef="bGwfMt7Q2BKArMVrAAAAAA==" TravelOrder="3" ProviderSegmentOrder="3" OptionalServicesIndicator="false" LinkAvailability="true" ElStat="A">
            <air:FlightDetails Key="bGwfMt7Q2BKAnMVrAAAAAA==" Origin="CDG" Destination="FCO" DepartureTime="2020-02-15T18:20:00.000+01:00" ArrivalTime="2020-02-15T20:25:00.000+01:00" FlightTime="125" TravelTime="125" Equipment="321" OriginTerminal="2F" DestinationTerminal="1" AutomatedCheckin="false" ElStat="C"/>
            <air:Connection Duration="60"/>
            <common_v47_0:SellMessage>DEPARTS CDG TERMINAL 2F - ARRIVES FCO TERMINAL 1</common_v47_0:SellMessage>
          </air:AirSegment>
          <air:AirSegment Key="XWjoMt3R2BKAO/goAAAAAA==" Group="1" Carrier="AZ" CabinClass="Economy" FlightNumber="210" ProviderCode="1V" Origin="FCO" Destination="LHR" DepartureTime="2020-02-15T21:25:00.000+01:00" ArrivalTime="2020-02-15T23:05:00.000+00:00" TravelTime="160" ClassOfService="X" ETicketability="Yes" Equipment="32S" MarriageGroup="2" Status="HK" ChangeOfPlane="false" GuaranteedPaymentCarrier="No" ProviderReservationInfoRef="bGwfMt7Q2BKArMVrAAAAAA==" TravelOrder="4" ProviderSegmentOrder="4" OptionalServicesIndicator="false" LinkAvailability="true" ElStat="A">
            <air:FlightDetails Key="bGwfMt7Q2BKAoMVrAAAAAA==" Origin="FCO" Destination="LHR" DepartureTime="2020-02-15T21:25:00.000+01:00" ArrivalTime="2020-02-15T23:05:00.000+00:00" FlightTime="160" TravelTime="160" Equipment="32S" OriginTerminal="1" DestinationTerminal="4" AutomatedCheckin="false" ElStat="C"/>
            <common_v47_0:SellMessage>OFFER CAR/HOTEL    >CAL	     >HOA</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>DEPARTS FCO TERMINAL 1  - ARRIVES LHR TERMINAL 4</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>ADD ADVANCE PASSENGER INFORMATION SSRS DOCA/DOCO/DOCS</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>PERSONAL DATA WHICH IS PROVIDED TO US IN CONNECTION</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>WITH YOUR TRAVEL MAY BE PASSED TO GOVERNMENT AUTHORITIES</common_v47_0:SellMessage>
            <common_v47_0:SellMessage>FOR BORDER CONTROL AND AVIATION SECURITY PURPOSES</common_v47_0:SellMessage>
          </air:AirSegment>
        </air:AirReservation>
        <common_v47_0:AgencyInfo>
          <common_v47_0:AgentAction ActionType="Created" AgentCode="uAPI4507433954-7b08af9f" BranchCode="P7118380" AgencyCode="S7105883" EventTime="2019-06-13T10:53:14.008+00:00"/>
        </common_v47_0:AgencyInfo>
      </universal:UniversalRecord>
    </universal:AirCreateReservationRsp>
  </SOAP:Body>
</SOAP:Envelope>
```
Any idea on what could be causing this?

### PR DESCRIPTION
**Pull request by creduo**
This PR should fix #412 

com:HostToken was missing in AirPricingSolution_XML.

### CODE REVIEW

File: src/Services/Air/AirParser.js [hunk 1]
```diff
@@ -249,7 +249,10 @@ function airPriceRspPricingSolutionXML(obj) {
   pricingSolution['air:AirPricingInfo'] = pricingInfos;
   const resultXml = {};
 
-  ['air:AirSegment', 'air:AirPricingInfo', 'air:FareNote'].forEach((root) => {
+  ['air:AirSegment', 'air:AirPricingInfo', 'air:FareNote', `common_${this.uapi_version}:HostToken`].forEach((root) => {
```
Comment (shmuga) [inline src/Services/Air/AirParser.js]: Are we sure this change fixes the bug? And why we need check for non existing tag?

Comment (creduo) [reply src/Services/Air/AirParser.js]: Some carriers return HostToken, some doesn't. 
If we don't check for non existing tag, following code may generate empty HostToken_XML tag which will break following RQ.

Comment (dchertousov) [reply src/Services/Air/AirParser.js]: @creduo kindly waiting for the tests.

Also should we just skip the solution, or should we each solution bookable in a way?

Comment (creduo) [reply src/Services/Air/AirParser.js]: @dchertousov Added tests for this case, 
and I don't follow what do you mean by solution. Is it air:AirPricingSolution?

**Comment by rickithadi** [comment]
Guess we are waiting on the review to merge? This seems to work for my use case and tests seem adequate

**Pull request merged**